frontend/packages/console-app/src/components/network-policies/network-policy-peer-selectors.tsx

```diff
@@ -18,18 +18,18 @@ export const NetworkPolicyPeerSelectors: React.FunctionComponent<PeerSelectorPro
   if (direction === 'ingress') {
     helpTextPodSelector = namespaceSelector
       ? t(
-          'console-app~If no pod selector is provided, traffic from all pods in eligible namespaces will be allowed.',
+          'console-app~If no pod selector is provided, traffic from all pods in eligible projects will be allowed.',
```

Comment:
It's going to be a lot of work putting in checks for the OpenShift flag everywhere :( We might consider that as a separate PR if we really want it.

Reply:
yes, so I've rolled back to using 'namespace'